homeassistant/components/uptime/sensor.py

```diff
@@ -48,7 +60,8 @@ class UptimeSensor(SensorEntity):
     _attr_device_class = SensorDeviceClass.TIMESTAMP
     _attr_should_poll = False
 
-    def __init__(self, name: str) -> None:
+    def __init__(self, entry: ConfigEntry) -> None:
         """Initialize the uptime sensor."""
-        self._attr_name = name
+        self._attr_name = entry.title
```

Comment:
Now there will be multiple places to set the name -> edit config entry and edit entity. That might be confusing?

Reply:
It will inherit, but can still be overridden. I think this pattern is very helpful actually (and used in more places nowadays).

This mainly works on config entries that have a single device or service.